config/clusters/callysto/common.values.yaml

```diff
@@ -106,6 +106,7 @@ jupyterhub:
             return None
 
         c.JupyterHub.authenticator_class = EmailAuthenticatingCILogonOAuthenticator
+        c.KubeSpawner.environment = {'PLOTLY_RENDERER': 'notebook_connected'}
```

Comment:
I would suggest using `singleuser.extraEnv` instead.

Reply:
Alternatively, you can also set this in your image! That *may* be a better fit, given this is probably tied to versions of plotly?

Reply:
Completely forgot that was an option! I've updated the branch. There is a group on there today so I don't want to change anything, I'll probably wait till Friday to merge if that is OK.